### PR TITLE
Add support for new types like: Seq and java.util.UUID, and define a …

### PR DESCRIPTION
**Pull request by ccojocar**
…new annotation which allows the insertion of comments as desciption keywords for case classes and their parameters

### CODE REVIEW

**Comment by saeta** [comment]
@cosmin-cojocar thank you for your contributions. This pull request got buried in my inbox. I will try and take a look presently. Thank you for your patience!

File: src/main/scala/org/coursera/autoschema/AutoSchema.scala [hunk 1]
```diff
@@ -16,9 +16,7 @@
 
 package org.coursera.autoschema
 
-import play.api.libs.json.JsArray
-import play.api.libs.json.Json
-import play.api.libs.json.JsObject
+import play.api.libs.json.{JsString, JsArray, Json, JsObject}
```
Comment (saeta) [inline src/main/scala/org/coursera/autoschema/AutoSchema.scala]: Coursera's Scala style guide is to put these each on separate lines. (This makes PR's cleaner when adding and removing imports.) Could you make this change?

**Comment by saeta** [comment]
In general, the change looks pretty good. One important request: could you please add some tests to your PR so we can prevent regressions in the future?

Thanks!

**Comment by ccojocar** [comment]
I removed the license header from build.sbt because sbt version 13.x complains about multiline comment.

**Comment by ccojocar** [comment]
I would really appreciate any feedback from you. Thanks

**Comment by saeta** [comment]
hey @cosmin-cojocar I think this look great. Thanks for the ping. I'm merging this presently.

**Comment by saeta** [comment]
Thank you very much @cosmin-cojocar for your contribution!

**Pull request merged**